code/solutions/architecting-templates/application-blue-green.template

```diff
@@ -0,0 +1,275 @@
+AWSTemplateFormatVersion: "2010-09-09"
+
+Description: Sample template describing an example application deployment. (uksb-1q9p31idr)
+
+Parameters:
+  AppNameTagValue:
+    Description: 'Please specify a value; minimum length: 3, maximum: 50.'
+    Type: String
+    Default: AWS CloudFormation Workshop - example app
+    MaxLength: 50
+    MinLength: 3
+
+  Env:
+    Description: The name of the environment name to use.
+    Type: String
+    AllowedValues:
+      - dev
+      - qa
+      - prod
+    Default: dev
+
+  HostedZoneStackName:
+    Description: The name of the CloudFormation stack you created for hosted zone resource(s).
+    Type: String
+    Default: cloudformation-workshop-dev-hosted-zone
+    AllowedPattern: ^[a-zA-Z]{1}[a-zA-Z0-9-]*$
+    MaxLength: 128
+    MinLength: 1
+
+  InstanceType:
+    Description: Amazon EC2 instance type to use for your instances.
+    Type: String
+    AllowedValues:
+      - t2.micro
+      - t2.small
+      - t2.medium
+      - t3.micro
+      - t3.small
+      - t3.medium
```

Comment:
```suggestion
      - t2.micro
      - t3.micro
```

Reply:
I would stick to 12 month free instances here.

Reply:
The `t2.micro` instance is the default, and here I'm just giving the opportunity to choose something else if needed.